pkg/ui/tab.go

```diff
@@ -100,8 +99,17 @@ func RemoveSlide(s int, tui *TUI) {
 }
 
 // Adds a slide to the end of currently present slides
-func AddSlide(tui *TUI) {
-	tabSlide := *NewTab("bash", os.Getenv("SHELL"), tui)
+func AddNewSlide(tui *TUI, name string, command string, args []string, isCluster bool) {
+	if isCluster {
+		for i, tab := range tui.TerminalTabs {
+			if tab.title == args[0] {
+				tui.TerminalPageBar.Highlight(strconv.Itoa(i)).
+					ScrollToHighlight()
+				return
+			}
+		}
+	}
+	tabSlide := *NewTab(name, command, args, tui)
```

Comment:
Why is this assigned to a pointer?
Generally, explicit pointer assignment isn't a best practice.

Reply:
Removed the pointer assignment